primitives/src/traits/zeitgeist_multi_reservable_currency.rs

```diff
@@ -29,13 +29,13 @@ pub trait ZeitgeistAssetManager<AccountId>: NamedMultiReservableCurrency<Account
     /// If the `currency_id` is the native currency, then return None.
     fn accounts_by_currency_id(
         currency_id: Self::CurrencyId,
-    ) -> Result<(usize, Vec<(AccountId, AccountData<Self::Balance>)>), DispatchError>;
+    ) -> Result<(usize, Vec<AccountId>), DispatchError>;
```

Comment:
When you delete this function, it's much easier to see, if you really looked at every code dependency references. (If the overall plan is it to remove this, as I think it is).

Reply:
I am using this in test.

Reply:
```bash
grep -rI --exclude-dir="./target" "accounts_by_currency_id" *
```